src/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixer.php

```diff
@@ -90,9 +90,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 $optionalTypeRegEx = $annotation->supportTypes()
                     ? sprintf('(?:%s\s+(?:\$\w+\s+)?)?', preg_quote(implode('|', $annotation->getTypes())))
                     : '';
-                $content = preg_replace_callback('/^(\s*\*\s*@\w+\s+'.$optionalTypeRegEx.')(.*)$/', function (array $matches) {
-                    return $matches[1].lcfirst($matches[2]);
-                }, $startLine->getContent(), 1);
+                $content = preg_replace_callback(
+                    '/^(\s*\*\s*@\w+\s+'.$optionalTypeRegEx.')(\p{Lu}?(?=\p{Ll}|\p{Zs}))(.*)$/',
+                    function (array $matches) {
+                        return $matches[1].strtolower($matches[2]).$matches[3];
```

Comment:
maybe it is better to test if `mb_strtolower` is available and use it if it is? (not sure so like to hear other opinions on this and I know it is a bit of feature creep in this PR)